source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -850,33 +850,30 @@ are parsed from the ismaster response in the obvious way.
 Server Description Equality
 ```````````````````````````
 
-For the purposes of updating topology description and publishing SDAM events,
-two server descriptions having the same address
-MUST be considered equal if and only if the values of
-`ServerDescription`_ fields marked (=) are respectively equal.
+For the purpose of determining whether to publish SDAM events, two server
+descriptions having the same address MUST be considered equal if and only if
+the values of `ServerDescription`_ fields marked (=) are respectively equal.
+Two topology descriptions MUST be considered equal if and only if they have
```

Comment:
I think this last sentence is redundant. I propose that we leave the previous language as-is and modify the equality description to better define what the comparison should be for the "set of server addresses" (I think you mean `hosts', 'passives', and 'arbiters' when you say "set of server addresses", if that's incorrect then we should fix that as well)

Reply:
Also, a lot of places in the spec refer to the definition of server description equality.  Let' not add a sentence to this section that is not relevant to those references.  If we need the sentence, let's add a new heading `Topology Description Equality`.  But let's only do that if there is some other section that needs to reference that definition.  Looking at the companion document titled `SDAM Monitoring Specification`, I don't see anything about when one would publish a TopologyDescriptionChangedEvent.  Should it be added?

Reply:
I removed the language relating to topology description equality.